modules/api/src/main/java/org/apache/ignite/lang/IndexNotFoundException.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.ignite.lang;
+
+/**
+ * Exception is thrown when appropriate index is not found.
+ */
+public class IndexNotFoundException extends IgniteException {
+    /**
+     * Create a new exception with given index name.
+     *
+     * @param name Column name.
+     * @param fullName Schema and table name.
+     */
+    public IndexNotFoundException(String name, String fullName) {
```

Comment:
```suggestion
     * @param indexName Index name.
     * @param tableName Table canonical name.
     */
    public IndexNotFoundException(String indexName, String tableName) {
```
I think it still not correct because, regarding the SQL standard, index has it's own canonical name, which doesn't contain table name. 
I wonder the index is top-level entity, but that's it.

Maybe the correct way is left just one parameter name, but mention that a full canonical name is expected here.